doc/user_manual/PostProcessors/SparseSensingdocumentation.tex

```diff
@@ -0,0 +1,60 @@
+    \item Brunton, Bingni W., et al. ``Sparse sensor placement optimization for classification.'' SIAM Journal on Applied Mathematics 76.5 (2016): 2099-2122 for classification and,
+    \item de Silva, Brian M., et al. ``PySensors: A Python package for sparse sensor placement.'' arXiv preprint arXiv:2102.13476 (2021) contains a full literature review along with examples and additional tips for using PySensors effectively.
+\end{itemize} 
+
+Some important terms related to sensor placement problems are : 
+
+\begin{itemize}
+    \item \textbf{Reconstruction} deals with predicting the values of a quantity of interest at different locations other than those where sensors are located. For example, one might predict the temperature at a point point in the middle of a fuel rod based on readings taken at various other positions. 
+
+    \item \textbf{Classification} is the problem of predicting which category an example belongs to, given a set of training data (e.g. determining whether digital photos are of dogs or cats).
+
+    \item \textbf{Bases} in which measurement data are represented can have a dramatic effect on performance. PySensors implements the three bases most commonly used for sparse sensor placement: raw measurements, SVD/POD/PCA modes, and random projections. These bases can be easily incorporated into either classification or reconstruction tasks. 
+\end{itemize}
+
+\ppType{SparseSensing post-processor}{SparseSensing}
+\begin{itemize}
+	\item \xmlNode{Goal}, The goal of the sparse sensor optimization (i.e., reconstruction or classification) should be filled in subType.
+	\item \xmlNode{TrainingData}, The Dataobject containing the training data
+	\item \xmlNode{features}, Features/inputs of the data model
+	\item \xmlNode{target}, target of data model
+	\item \xmlNode{basis}, The type of basis onto which the data are projected
+	\item \xmlNode{nModes}, The number of modes retained
+	\item \xmlNode{nSensors}, The number of sensors used
+	\item \xmlNode{threshold}, The percentage of sensors used (i.e., nSensors/nFeatures)
+	\item \xmlNode{optimizer}, The type of optimizer used
+\end{itemize}
+
+\textbf{Example:}
+\begin{lstlisting}[style=XML]
+<Simulation>
+   ...
+   <Models>
+      ...
+      <PostProcessor name="mySPSL" subType="SparseSensing" verbosity="debug">
+      <Goal subType="reconstruction">
+        <TrainingData>myDO</TrainingData>
+        <features>x,y</features>
+        <target>v</target>
+        <basis>svd</basis> <!--Ddefault: svd-->
+        <nModes>5</nModes> <!--default: opt, allows the algorithm to pick nModes-->
+        <nSensors>5</nSensors><!--default: opt, allows the algorithm to pick nSensors-->
+        <threshold>0.2</threshold>
```

Comment:
I think Threshold is no longer there. @niharika2999, please remove it for now.

Reply:
Please change this test to the latest one from my latest PR. https://github.com/idaholab/raven/blob/e6972cf2e762fa8a6b4da8cafe96516482f33c2b/tests/framework/PostProcessors/SparseSensing/testSPSLOptiTwist.xml